nodejs/week2/homework/meal-sharing/src/backend/api/meals-router.js

```diff
@@ -0,0 +1,39 @@
+const express = require("express");
+const router = express.Router();
+
+const meals = require("./../data/meals.json");
+
+router.get("/", async (request, response) => {
+  let maxPrice = request.query.maxPrice;
+  let title = request.query.title;
+  let createdAfter = request.query.createdAfter;
+  let limit = request.query.limit;
+  if (maxPrice) {
+    const resultForMaxPrice = meals.filter((meal) => meal.price < maxPrice)
+    response.send(resultForMaxPrice);
+  } else if (title) {
+    const resultForTitle = meals.filter((meal) => meal.title.toLocaleLowerCase().includes(title.toLocaleLowerCase())
+    )
+    response.send(resultForTitle)
+  } else if (createdAfter) {
+    const resultForCreatedAfter = meals.filter((meal) => meal.createdAt.valueOf > createdAfter)
+    response.send(resultForCreatedAfter)
+  }
+  else {
```

Comment:
How about the limit parameter? Take a look at slice method of an array as a possible way to implement it.